easybuild/easyconfigs/l/libxml2/libxml2-2.9.1-intel-2014b-Python-2.7.8.eb

```diff
@@ -0,0 +1,32 @@
+name = 'libxml2'
+version = '2.9.1'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and toolchain developed for the Gnome project (but usable
+ outside of the Gnome platform)."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic'
```

Comment:
`$CCFLAGS` not being picked up (everywhere)?

doesn't hurt to add the toolchain option, but the `--with-pic` likely has to stay